modules/standard/Map.chpl

```diff
@@ -850,7 +957,23 @@ module Map {
     :rtype: `bool`
   */
   operator map.==(const ref a: map(?kt, ?vt, ?ps),
-                  const ref b: map(kt, vt, ps)): bool {
+                  const ref b: map(kt, vt, ps)): bool throws
+    where mapMethodsThrow{
+    for key in a.keys() {
+      if !b.contains(key) || a[key] != b[key] then
```

Comment:
Shouldn't the `contains` check ensure that this method never throws?

Reply:
That is a good point, I've updated to use `try!` instead, but let me know if you had something else in mind. (just noting that I still need to update the `.good` files to reflect that and am working on that now)

Reply:
That's what I was thinking, thanks!